### DISCUSSION TITLE
Pinned version of Paste for Flex is not mentioned on NPMJS package version

### DISCUSSION THREAD

**Discussion by harvard1807** [twilio-labs/paste#3959] [General] [answered]
Hello Team,
As of now pinned version of Paste in Flex 2.X is 15.3.6 but this version is not mentioned here https://www.npmjs.com/package/@twilio-paste/core?activeTab=versions. Can someone please advise. Thanks

**Comment by harvard1807** [comment]
Hello Team,
Please advise on this. Thanks

**Reply by krisantrobus** [reply, marked as answer]
Hi Harsh,

Apologies for the delay in response, we had to chase teams internally. You are correct, 15.3.6 is not a version that exists. I reached out with our Flex team to confirm and got the response:

```
Flex UI 2.7 has "@svc.tw-paste-github/core": "^15.3.1"
```

This maps to Paste [15.3.1](https://www.npmjs.com/package/@twilio-paste/core/v/15.3.1) and is listed in npm.
